modules/ListView.js

```diff
@@ -295,7 +306,6 @@
           // Shouldn't get here. Item view is missing...
```

Comment:
I think that may be unnecessarily jarring. For whatever reason a view not existing to a model in modelsToRender is not necessarily a programmer error - so I'd rather not throw a thread-ending error.